docs/design/index.md

```diff
@@ -40,8 +40,25 @@ The downside of the microautonomy architecture is that the computational perform
 
 ### Autoware concepts
 
+The concept of the Autoware is available [here](autoware-concepts/index.md).
+
+- Target readers:
+- Purpose:
```

Comment:
If it's empty, to avoid confusion:

```suggestion
TODO(TIER IV): Add target readers and purpose.
```

Reply:
Of course, it's better to write something.

@mitsudome-r @LalithVipulananthan Any ideas? Other sections are written as the attached image.

![image](https://user-images.githubusercontent.com/31987104/167606074-97a49869-d726-4dea-991a-80a22a824079.png)

Reply:
I think we can delete it for now.
https://github.com/autowarefoundation/autoware-documentation/pull/86/files#r869934472

Reply:
```suggestion
```

Reply:
Addressed.